test/onnx/test_verify.py

```diff
@@ -45,6 +47,7 @@ def forward(self, x, y):
 
         x = torch.tensor([1, 2])
         y = torch.tensor([3, 4])
+        torch.onnx.symbolic_registry.register_op('BrokenAdd', BrokenAdd.symbolic, '', 9)
```

Comment:
Why do we need to call register_op here?

Reply:
The symbolic function used here is not registered since it is not in symbolic_opset9/10.py.
So registering the op is required. We could maybe use register_custom_op_symbolic() in utils.py to do it?

Reply:
This was actually not needed by modifying onnx.cpp. 
Pushing the change shortly :)